src/Command.js

```diff
@@ -214,6 +214,7 @@ export default class Command {
     return new Promise((resolve, reject) => {
       const onComplete = (err, exitCode) => {
         if (err) {
+          if(typeof err === 'string') err = { stack:err };
```

Comment:
Agreed, but when I was running tests, that's what I was seeing. The Promises were stalling out, and after digging through the code I found that the errors weren't being reported correctly because this bit of the code was expecting an Object to set `exitCode` on. This check doesn't hurt anything, and makes it work in such cases.